### PR TITLE
Update alias and alias_method chapters

### PR DESCRIPTION
**Pull request by mshakhan**

### CODE REVIEW

File: README-ruRU.md [hunk 1]
```diff
@@ -2430,11 +2430,10 @@
 
 <!--- @FIXME -->
```
Comment (arbox) [inline README-ruRU.md]: Please remove the tag.

File: README-ruRU.md [hunk 2]
```diff
@@ -2430,11 +2430,10 @@
...
 * <a name="alias-method-lexically"></a>
-  Prefer `alias` when aliasing methods in lexical class scope as the
-  resolution of `self` in this context is also lexical, and it communicates
-  clearly to the user that the indirection of your alias will not be altered
-  at runtime or by any subclass unless made explicit.
-<sup>[[link](#alias-method-lexically)]</sup>
+  Используйте `alias` при определении алиасов методов в лексической области видимости класса. 
+  `self` в данном случае также имеет лексическую область видимости, и это подчеркивает тот факт, что
+  алиас будет указывать на метод того класса, в котором определен. 
+  Вызов не будет перенаправлен неявно.<sup>[[link](#alias-method-lexically)]</sup>
```
Comment (arbox) [inline README-ruRU.md]: Keep <sup> on a separate line.

File: README-ruRU.md [hunk 3]
```diff
@@ -2430,11 +2430,10 @@
 
 <!--- @FIXME -->
 * <a name="alias-method-lexically"></a>
-  Prefer `alias` when aliasing methods in lexical class scope as the
-  resolution of `self` in this context is also lexical, and it communicates
-  clearly to the user that the indirection of your alias will not be altered
-  at runtime or by any subclass unless made explicit.
-<sup>[[link](#alias-method-lexically)]</sup>
+  Используйте `alias` при определении алиасов методов в лексической области видимости класса. 
+  `self` в данном случае также имеет лексическую область видимости, и это подчеркивает тот факт, что
```
Comment (arbox) [inline README-ruRU.md]: Keep lines short (normally up to 80 characters).

File: README-ruRU.md [hunk 4]
```diff
@@ -2462,10 +2461,9 @@
   end
   ```
 
-  In this example, `Fugitive#given_name` would still call the original
-  `Westerner#first_name` method, not `Fugitive#first_name`. To override the
-  behavior of `Fugitive#given_name` as well, you'd have to redefine it in the
-  derived class.
+  В этом примере, `Fugitive#given_name` будет вызывать метод базовго класса
```
Comment (arbox) [inline README-ruRU.md]: Comma is superfluous.

**Comment by arbox** [comment]
Thank you :thumbsup: Please incorporate comments in the same commit and repush it. I'll merge your contribution.

**Comment by mshakhan** [comment]
Done!

**Comment by arbox** [comment]
Good job, thank you :thumbsup:

**Pull request merged**